prow/cmd/deck/main.go

```diff
@@ -373,7 +373,7 @@ func handleConfig(ca configAgent) http.HandlerFunc {
 		// TODO(bentheelder): add the ability to query for portions of the config?
 		w.Header().Set("Cache-Control", "no-cache")
 		config := ca.Config()
-		b, err := yaml.Marshal(config)
+		b, err := json.Marshal(config)
```

Comment:
I'd still like a way to expose the yaml so we can easily compare the config file currently loaded to the config stored elsewhere.

Reply:
Added a separate endpoint for JSON - let me know if that's ok with you.

Reply:
thanks!